docs/Text.md

```diff
@@ -0,0 +1,178 @@
+# Adding Text
```

Comment:
This PR contains commits that are part of https://github.com/PyFPDF/fpdf2/pull/363 ?

Reply:
I started that here and later decided to make it a separate PR.
Will remove the leftovers here.